src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,147 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public void add(T value) {
+        tail = new Node<>(tail, value, null);
+        if (head == null) {
+            head = tail;
+        } else {
+            tail.prev.next = tail;
+        }
+        size++;
```

Comment:
```suggestion
        if (head == null) {
            head = node;
        } else {
            node.prev = tail;
            tail.next = node;
        }
        
```

Reply:
and after that 
assign node to the tail 
 tail = node;
 and increment the size